lib/internal/child_process.js

```diff
@@ -269,6 +269,24 @@ function closePendingHandle(target) {
   target._pendingMessage = null;
 }
 
+function clearHandleQueue(target) {
+  const queue = target._handleQueue;
+  target._handleQueue = null;
+
+  const ex = new ERR_IPC_CHANNEL_CLOSED();
+
+  for (var i = 0; i < queue.length; i++) {
```

Comment:
`var` → `let`

Reply:
@tniessen we have a explicit eslint rule to prevent using `let` in loops.

Reply:
If I recall correctly (a big "if"), we left the rule there to ease backporting. I recall deciding that the rule could be removed once it was no longer a problem in the V8 version in the oldest supported release line. `let` in a loop is still slower than `var` in Node.js 6.14.2 (latest 6.x.x release). It's not there in Node.js 8.11.1 (latest 8.x.x release).

Reply:
Okay, thanks for the explanation, I remember talking to @fhinkel about this concerning *recent* v8 releases.

Reply:
See https://github.com/nodejs/node/pull/15648#issuecomment-333030885